packages/vendure-plugin-public-customer-groups/src/public-customer-groups.resolver.ts

```diff
@@ -0,0 +1,36 @@
+import { Parent, ResolveField, Resolver } from '@nestjs/graphql';
+import {
+  Ctx,
+  RequestContext,
+  Customer,
+  CustomerService,
+  CustomerGroup,
+  PaginatedList,
+} from '@vendure/core';
+
+@Resolver()
+export class PublicCustomerGroupsResolver {
+  constructor(private readonly customerService: CustomerService) {}
+
+  @ResolveField('customerGroups')
+  @Resolver('Customer')
+  async customerGroups(
+    @Ctx() ctx: RequestContext,
+    @Parent() customer: Customer
+  ): Promise<CustomerGroup[]> {
+    const customerGroups = await this.customerService.getCustomerGroups(
+      ctx,
+      customer.id
+    );
+    return customerGroups.filter((group) => !!group.customFields.isPublic);
+  }
+
+  @ResolveField('customers')
+  @Resolver('CustomerGroup')
+  async customers(
+    @Ctx() ctx: RequestContext,
+    @Parent() _: CustomerGroup
+  ): Promise<PaginatedList<Customer>> {
+    return { items: [], totalItems: 0 };
```

Comment:
Can you add a commen here? Something like `Resolve group.customers to emoty array, to prevent exposure of other customers in the Shop API`
Sidequestion: What happens without this resolver? Does Vendure actually resolve the list of customers, or does it return null?